docs/reference/migration/migrate_8_0/search.asciidoc

```diff
@@ -6,6 +6,25 @@
 //Installation and Upgrade Guide
 
 //tag::notable-breaking-changes[]
+[[msearch-empty-line-support]]
+.The multi search API now parses an empty first line as action metadata in text files.
+[%collapsible]
+====
+*Details* +
+The multi search API now parses an empty first line as empty action metadata
+when you provide the request body as a text file, such as when using curl's
+`--data-binary` flag.
+
+The API no longer supports text files that contain:
+
+* An empty first line followed by a line containing only `{}`.
+* An empty first line followed by another empty line.
```

Comment:
I originally included some example snippets here, but it looks like we strip out leading whitespace in snippets. ☹️ 
I couldn't figure out a good workaround. Let me know if this text is enough without a visual aid.